handlers/svg-open-street-map/osm-svg.py

```diff
@@ -75,186 +123,210 @@ def handle():
                     response_schema, resolver=resolver)
                 validator.validate(response)
             except jsonschema.exceptions.ValidationError as error:
-                logging.error(error)
+                LOGGER.error(error)
                 return jsonify("Invalid Preprocessor JSON format"), 500
-            logging.debug("Sending response")
+            LOGGER.debug("Sending response")
             return response
 
+        LOGGER.debug("Checking for OpenStreetMap map data ")
         if "ca.mcgill.a11y.image.preprocessor.openstreetmap"\
```

Comment:
Same as above...this is just an else statement for the following if.

Reply:
Well, I was only being careful not to add things that would be portrayed as new changes.  I've however made the changes now.